tests/onegov/form/test_grammar.py

```diff
@@ -26,7 +26,7 @@
     time,
     url,
     valid_date_range,
-    with_whitespace_inside,
+    with_whitespace_inside, video_url,
```

Comment:
```suggestion
    with_whitespace_inside,
    video_url,
```
Formatting nit, it's fine to have multiple imports on one line, but then you should reformat everything to remain consistent, so either everything on its own line or everything split roughly evenly among a fewer number of lines, although for avoiding merge conflicts it's usually better to have separate lines.